README.md

```diff
@@ -134,23 +145,36 @@ git add ruleset.xml phpmd.xml .eslintrc .stylelintrc grumphp.yml
 
 #### 2. Add following to project`s 'Makefile':
 ```makefile
-linters-init: # init linters on local machine
+### Code Quality section
+CQ_STARTING_COMMIT_HASH='a000z999'
+CQ_STARTING_COMMIT_DATE='01/01/2021'
+
+linters-init: ## init linters on local machine
 	warden env exec php-fpm chmod +x vendor/space48/magento2-code-quality/script/install.sh
 	warden env exec php-fpm ./vendor/space48/magento2-code-quality/script/install.sh
 	warden env exec php-fpm /bin/bash -c '[ -d .git ] || mkdir .git'
 	chmod +x vendor/space48/magento2-code-quality/script/add-hook.sh
 	vendor/space48/magento2-code-quality/script/add-hook.sh
 
-analyse: # analyses all code from starting commit hash to HEAD
-	git diff a000z999..HEAD | warden env run --rm php-fpm 'vendor/phpro/grumphp/bin/grumphp' run
+analyse: ## analyses all code from starting commit hash to HEAD
+	git diff ${CQ_STARTING_COMMIT_HASH}..HEAD | warden env run --rm php-fpm 'vendor/phpro/grumphp/bin/grumphp' run
+
+fix: ## analyses all code from starting commit hash to HEAD
+	git diff ${CQ_STARTING_COMMIT_HASH}..HEAD | warden env run --rm php-fpm 'vendor/phpro/grumphp/bin/grumphp' run --fix
 
-precommit: # analyses code staged for commit
+precommit: ## analyses code staged for commit
 	git diff --staged | warden env run --rm php-fpm 'vendor/phpro/grumphp/bin/grumphp' run
+
+analyse-ci: # Called during build on CI
+	git fetch --shallow-since=${CQ_STARTING_COMMIT_DATE}
+	git diff ${CQ_STARTING_COMMIT_HASH}..HEAD | vendor/phpro/grumphp/bin/grumphp run
 ```
 
 Replace the sample `a000z999` commit hash with the hash from the project where you want to start linting from.
 Files modified after the starting commit hash will be linted during project build and will fail the build on linter violations.
 
+Refer to `code-quality.mk` as a sample.
```

Comment:
(Off topic) This gave me an idea:

In Makefiles, you can include other makefiles in by adding:
```
include path/to/makefile.mk
```
So I wonder if our instructions should just be to add the required variables and the line above to the main project makefile, instead of each project maintaining their own copy of the code quality targets. Just a thought!